### ISSUE TITLE
Code Builder Guide is unreadable / fails accessibility test in Dark Mode

### ISSUE THREAD

**Issue by thedanfernandez** [forcedotcom/try-code-builder-feedback#137]
**Describe the bug**

Goal: Read the Code Builder Guide landing page text in dark mode

**To Reproduce**

1. Open Code Builder
2. From Code Builder, type <kbd>CMD</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> and type `Color Theme`. From the list, select `Preferences: Color Theme` and hit <kbd>enter</kbd>.  
3. Change the theme to `Dark+ Default mode`
4. Open the Code Builder Guide

**Expected behavior**

The content of the Code Builder page would be readable on either the dark or light theme, typically with a color contrast of ~4:1

**Screenshots**

![code-builder-guide](https://user-images.githubusercontent.com/8809372/161868450-6a5983f5-ecda-4c79-8124-1b95e04814d6.png)

Screenshot here, with red squares around hard-to-read text or images

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS Monterey
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] Version 100.0.4896.75 (Official Build) (x86_64)

**Additional context**

**Comment by jonnyhork** [comment]
This has been fixed  in https://github.com/forcedotcom/code-builder-vscode/pull/37
